bedrock/mozorg/templates/mozorg/about/webvision/full.html

```diff
@@ -0,0 +1,29 @@
+{#
+ This Source Code Form is subject to the terms of the Mozilla Public
+ License, v. 2.0. If a copy of the MPL was not distributed with this
+ file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#}
+
+{% extends "mozorg/about/webvision/summary.html" %}
+
+{% block content %}
+  <main role="main" class="c-webvision-article-full mzp-l-content">
+    {% block side_nav %}
```

Comment:
I think this side_nav block should have been outside of the content block, but it was refusing to render for me unless i placed it inside.

Reply:
There's no `side_nav` block in the base template you're extending, so this is defining a new block in this template alone. It won't render outside of `content` because jinja doesn't know where to render the block without it being defined further up the chain.

I think you don't need it to be a template block at all and just it's fine as just a `nav` right here.